php/class-connect.php

```diff
@@ -456,7 +456,7 @@ public function check_status() {
 		$status = $this->test_ping();
 		$this->settings->get_setting( 'status' )->save_value( $status );
 
-		return $status;
+		return is_wp_error( $status ) ? $status->get_error_message() : $status['status'];
```

Comment:
@DavidCramer 

This change will require a change in the docblock for this method, right?